spec/pandadoc/api/params_validator_spec.rb

```diff
@@ -11,10 +11,10 @@
     describe 'passes validations' do
       describe 'invalid parameter' do
         it 'returns only the valid params' do
-          params = { valid: '1', invalid: '2' }
-          validations = { valid: { required: false, type: String } }
+          params = { valid: '1', invalid: '2', images: [{ name: 'Image1', urls: ['https://google.com'] }] }
```

Comment:
The tests in `/document_spec.rb` didn't seem to really be testing much so I added this test since it actually gave a test failure when the data was wrong